cmd/src/server.rs

```diff
@@ -172,7 +174,14 @@ impl<ER: RaftEngine> TiKVServer<ER> {
             SecurityManager::new(&config.security)
                 .unwrap_or_else(|e| fatal!("failed to create security manager: {}", e)),
         );
-        let pd_client = Self::connect_to_pd_cluster(&mut config, Arc::clone(&security_mgr));
+        let env = Arc::new(
+            EnvBuilder::new()
+                .cq_count(config.server.grpc_concurrency)
```

Comment:
We have meet some situations that PD worker overloads if there are too many regions. So we need to increase the pool size. And Considering users that has adjusted this value, I think we can use `config.sever.grpc_concurrency + CQ_COUNT` here.

Reply:
I do not think so. CQ_COUNT is 1 and most of case it is enough. Because pd-worker do more calculate jobs than pd-client thread, if one thread is not able to deal with heartbeats of regions, neither is pd-worker.
Even if we need more grpc thread to deal with pd messages, the better solution is adjust the value of `grpc_concurrency`.

Reply:
Or we can set default value of grpc_concurrency to `5`.